pkgs/servers/web-apps/fresh-rss/default.nix

```diff
@@ -0,0 +1,37 @@
+{ stdenv, fetchFromGitHub, writeText }:
+
+stdenv.mkDerivation rec {
+  pname = "fresh-rss";
+  version = "1.15.3";
+
+  src = fetchFromGitHub {
+    owner = "FreshRSS";
+    repo = "FreshRSS";
+    rev = version;
+    sha256 = "0xnflvqhifk0icqqclv4qbs43x77syi3szha5mzqxf2fxyzrs358";
+  };
+
+  phpConfig = writeText "constants.local.php" ''
+  <?php
+    safe_define('DATA_PATH', '/var/lib/fresh-rss');
```

Comment:
It'd be nice to tie this together with the `stateDir` variable defined in the module. Or is there a need to change the directories independently from each other in the package and module?

Reply:
It is not really configurable in the module either so this should be fine.

Reply:
My intention is to leave this hardcoded. Whether a module has a configurable state directory or not is left up to the discretion of the module author. It is more work to make the state directory configurable, so I went with not.

Reply:
It's just alway a potential source of issues if you define the same constant at multiple places and then forget to change both at some point.
But yeah, this is just a minor point.